policies/ip-policy.md

```diff
@@ -28,54 +24,51 @@ The CF is most familiar with the following four licenses, which can simplify the
 > [!TIP]
 > Projects can pick appropriate licenses, including Creative Commons licenses that permit commercial use, for their non-code content like documentation and images.
 
-A CF Project can define a "Project Documentation License" to cover all documentation, image, and audiovisual files (e.g., .txt., .rtf, .doc, .pdf, .jpg, .tif, .mp3, .wav, and some .html files) (including without limitation code that is intended as sample code if included in a documentation file).
+A CF Project may define a "Project Documentation License" to cover all documentation, image, and audiovisual files (e.g., .txt., .rtf, .doc, .pdf, .jpg, .tif, .mp3, .wav, and some .html files) (including without limitation code that is intended as sample code if included in a documentation file).
 The Project Documentation License can be any OSI-approved license or a Creative Commons license (like [CC BY 4.0](http://creativecommons.org/licenses/by/4.0/)) that permits commercial use.
 
 ## Contributions
 
-> [!TIP]
-> All code contributions must align with the project’s license and come with a promise (DCO) from contributors that they have the right to contribute under these terms.
+All new code contributions to any Project must be made under the Project Code License.
 
-All new code contributions to any Project shall be made under the Project Code License accompanied by a Developers Certificate of Origin (DCO, available at <http://developercertificate.org/>), which will bind the individual contributor and, if applicable, their employer to the Project Code License.
+Contributions must be accompanied by either a Developer Certificate of Origin (DCO available at <http://developercertificate.org/>) or a Contributor License Agreement (CLA), as determined by the project’s technical governing body.
 
-The technical governing body of each Project may provide for additional requirements with respect to contributions. For example, a Project may require that new code contributions to a Project be accompanied by a signed Contributor License Agreement (CLA).
+The DCO or CLA binds the individual contributor and, if applicable, their employer to the Project Code License.
 
 ### Developers Certificate of Origin (DCO) or Contributor License Agreement (CLA)?
 
 The DCO and CLA differ in their approach to intellectual property rights.
 
-- A DCO is a straightforward way for contributors to assert their rights to submit work, typically through a declaration in the commit message.
-- A CLA is a formal agreement that allows contributors to grant broader rights over their contributions to the project.
+- A DCO is a straightforward method for contributors to confirm they have the necessary rights to submit work, typically through a declaration in the commit message.
```

Comment:
```suggestion
- A DCO is a straightforward method for contributors to declare that they have the necessary rights to submit work, typically through a declaration in the commit message.
```

Reply:
`confirm` was used on recommendation from our lawyers. So while I agree that affirm or declare would make sense, I'll follow guidance in this case.